pylint/pyreverse/mermaidjs_printer.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2021 Andreas Finkler <andi.finkler@gmail.com>
```

Comment:
I think your name is the one that should appear here! 😊

Reply:
You can remove the copyright it will be automatically generated before release :)